dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -225,28 +176,20 @@ public URIStatus next() {
    */
   public List<DoraLoadTask> prepareNextTasks() {
     LOG.debug("Preparing next set of tasks for jobId:{}", mJobId);
-    ImmutableList.Builder<URIStatus> batchBuilder = ImmutableList.builder();
+    ImmutableList.Builder<UfsStatus> batchBuilder = ImmutableList.builder();
     int i = 0;
     int startRetryListSize = mRetryFiles.size();
     int filesToLoad = 0;
     while (filesToLoad < RETRY_THRESHOLD
-        && i++ < startRetryListSize
-        && mRetryFiles.peek() != null) {
-      String path = mRetryFiles.poll();
-      try {
-        URIStatus uriStatus = mFs.getStatus(new AlluxioURI(path));
-        batchBuilder.add(uriStatus);
-        ++filesToLoad;
-      } catch (AlluxioException | IOException ex) {
-        if (!(ex instanceof FileDoesNotExistException)) {
-          mRetryFiles.offer(path);
-        }
-      }
+        && i++ < startRetryListSize && mRetryFiles.peek() != null) {
+      UfsStatus ufsStatus = mRetryFiles.poll();
+      batchBuilder.add(ufsStatus);
```

Comment:
when we get the ufsstatus from retry shall we re-fetch the status from ufs to make sure its not stale ?

Reply:
okay i'll revert